lib/constants.js

```diff
@@ -1,6 +1,21 @@
 'use strict'; // eslint-disable-line strict
 
+// The min value here is to manage further backward compat if we
+// need it
+const iamSecurityTokenSizeMin = 128;
+const iamSecurityTokenSizeMax = 128;
+// Security token is an hex string (no real format from amazon)
+const iamSecurityTokenSizePattern =
+    new RegExp(`^[a-f0-9]{${iamSecurityTokenSizeMin},` +
```

Comment:
not sure if its necessary to allow uppercases letters, since its only internal, we can generate what we want, but why not